Week3/Kemirdin-DBHW3-Books.sql

```diff
@@ -0,0 +1,143 @@
+CREATE DATABASE DBBooks;
+USE DBBooks;
+--    Database: Books
+-- ------------------------------------------------------
+-- Table structure for table `Authors`
+
+
+DROP TABLE IF EXISTS Authors;
+
+CREATE TABLE Authors
+(
+    author_id int(11) NOT NULL
+    AUTO_INCREMENT,
+  last_name text COLLATE utf8_unicode_ci NOT NULL,
+  first_name text COLLATE utf8_unicode_ci NOT NULL,
+  PRIMARY KEY
+    ( author_id )
+) ENGINE=InnoDB AUTO_INCREMENT=9 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+
+
+-- Dumping data for table `Authors`
+LOCK TABLES Authors WRITE;
+
+    INSERT INTO Authors
+    VALUES
+        (1, 'Lopez Baranda', 'Christina'),
+        (2, 'Jin-Soon', 'Sin'),
+        (3, 'Jones', 'Hannah'),
+        (4, 'Novak', 'Stanislaw'),
+        (5, 'Turay', 'Tandice'),
+        (6, 'Roy', 'Shanta'),
+        (7, 'Berger', 'Henry')
+    ,
+ 
+UNLOCK TABLES;
+    -- Table structure for table `Books`
+    DROP TABLE IF EXISTS Books;
+
+    CREATE TABLE Books
+    (
+        book_id int(11) NOT NULL
+        AUTO_INCREMENT,
+  title text COLLATE utf8_unicode_ci NOT NULL,
+  ISBN text COLLATE utf8_unicode_ci NOT NULL,
+  PRIMARY KEY
+        (book_id)
```

Comment:
Why do you need a book_id in addition to an ISBN? An ISBN is a unique identifier :smile: